### DISCUSSION TITLE
Lunar Spell Geomancy to sync with the Time Tracking tab?

### DISCUSSION THREAD

**Discussion by LevyWilson** [runelite/runelite#17240] [Ideas]
Geomancy is a Lunar spell that gives the player an overview of all their crops in the world, including what's protected, what fertilizer was used, if something is dead or diseased, and what's fully grown, of course. It's essentially a live version of the Time Tracking tab without needing to visit every plot. Would it be possible to have the Time Tracking tab sync with this spell when it is used? It would make it easy to keep track of what's dead to plan for plant resurrection spells.

**Comment by YvesW** [comment]
https://github.com/runelite/runelite/issues/7350

**Reply by LevyWilson** [reply]
Okay, the search bar is broken then, cus I freakin searched for Geomancy and nothing showed up.
